code/modules/antagonists/slaughter/slaughter.dm

```diff
@@ -111,8 +115,44 @@
 /mob/living/simple_animal/slaughter/phasein()
 	. = ..()
 	add_movespeed_modifier(/datum/movespeed_modifier/slaughter)
-	addtimer(CALLBACK(src, .proc/remove_movespeed_modifier, /datum/movespeed_modifier/slaughter), 6 SECONDS, TIMER_UNIQUE | TIMER_OVERRIDE)
+	var/slowdown_time = 6 SECONDS + (0.5 * consumed_buff)
+	addtimer(CALLBACK(src, .proc/remove_movespeed_modifier, /datum/movespeed_modifier/slaughter), slowdown_time, TIMER_UNIQUE | TIMER_OVERRIDE)
+
+/mob/living/simple_animal/slaughter/Destroy()
+	release_victims()
+	. = ..()
+
+/mob/living/simple_animal/slaughter/proc/release_victims()
+	if(!consumed_mobs)
+		return
+
+	for(var/mob/living/M in consumed_mobs)
+		if(!M)
+			continue
+		var/turf/T = find_safe_turf()
+		if(!T)
+			T = get_turf(src)
+		M.forceMove(T)
+
+/mob/living/simple_animal/slaughter/proc/refresh_consumed_buff()
+	melee_damage_lower = 22.5 + (0.5 * consumed_buff)
```

Comment:
`melee_damage_lower += (0.5 * consumed_buff)`
. with upper as well

Reply:
that would lead to a quadratic curve instead of a linear one, each heart increasing damage by (0.5*hearts) instead of just 0.5; I know having 22.5 as a magic number is ugly as sin, and I could probably just make it a define, but I wrote it this way mostly so I could make laughter demon buff work differently.